homeassistant/components/schlage/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
```

Comment:
Opinionated comment about the coordinator: There's no real need to wrap the data of the coordinator in the dataclass `SchlageData`. It could just do `dict[str, LockData]` raw